recipes/openmathreasoning/configs/solution_sdg/qwq.yaml

```diff
@@ -54,7 +66,7 @@ stages:
       server_gpus: 8
       server_nodes: 1
       num_random_seeds: ${num_random_seeds_to_generate}
-      num_chunks: 10  # since data is big, we are parallelizing it 10x (for each seed, so in total 80 jobs are scheduled)
+      # num_chunks: 10  # since data is big, we are parallelizing it 10x (for each seed, so in total 80 jobs are scheduled)
```

Comment:
why this change?

Reply:
Was creating too many jobs even when this toy dataset is quite small. I can change the comment to say "can use `num_chunks` to parallelize"

Reply:
Let me add it back. I removed it during my testing, and commented it for later use if users want to use it for larger datasets. But we can keep this as the default.